lessons/python/rainfall_runoff.ipynb

```diff
@@ -457,9 +467,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Ivy",
+   "display_name": "Python 3 (ipykernel)",
    "language": "python",
-   "name": "ivy"
+   "name": "python3"
```

Comment:
@gantian127 Could you revert these to Ivy/ivy, please?

Reply:
@mdpiper There should be a linter for this. Did you disable it? Was it not working correctly?

Reply:
@mcflugen I didn't see it in the latest *heartfelt-hooks*. Did the name change, or did I just miss it?

Reply:
It's not there anymore but it's still in my version. Maybe I just added it to a branch I was working on and never merged it. If you like, I can create a pull request that adds it. It should just be adding something like the following under the *heartfelt-hooks* section,
```yaml
  - id: nb-check-kernel
    args: [
      --verbose,
      --fix,
      --kernel-name=ivy,
      --kernel-display-name=Ivy,
      --kernel-language=python
    ]
```

Reply:
If you add it to _heartfelt-hooks_ & bump the release, I'll add the hook back to the Ivy `.pre-commit-config.yaml`.